conclusion.Rmd

```diff
@@ -1,5 +1,3 @@
-#  (PART) Conclusion {-}
```

Comment:
your chapter should live in "advanced topics" I think. It'd make the TOC clearer.

Reply:
done